docs/modules/commons/pages/expressions.adoc

```diff
@@ -323,6 +323,31 @@ Escapes reserved characters in https://www.w3schools.com/html/html_entities.asp[
 |`M&amp;Ms`
 |===
 
+=== `quotePattern`
```

Comment:
why is it needed? why users can't quote regex manually?

Reply:
We are using backend data as the expected  value. In our case contains won't fit because we need stricter checks. I am not sure about all the cases, maybe a "starts with" check might help